### DISCUSSION TITLE
[Feature] Location history map

### DISCUSSION THREAD

**Discussion by C9Glax** [immich-app/immich#1791] [Feature Request]
### The feature

This feature would - similar to **company-that-should-not-be-named** timeline feature - have location pins (maybe grouped within a radius) and then the ability to group photos to view them and create albums based on this group.

Also you could perhaps select a timeframe and view a map of the locations where photos were taken.

![grafik](https://user-images.githubusercontent.com/13404778/219874084-5b43ac1f-b797-4f69-9ace-d3a9130c4cc6.png)

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by EnochPrime** [comment]
Is this a duplicate of #1693?

**Reply by bo0tzz** [reply]
Yes
